### ISSUE TITLE
how to delete channel using custommix file (after HD swap)

### ISSUE THREAD

**Issue by chenks** [oe-alliance/AutoBouquetsMaker#126]
this isn't really an issue but more of a question.
how do i remove specific channels using a custommix file?

example, if wanted to remove the SD versions of channels where an HD channel exists already (and is already swapped using ABM hd swap) ?

if i wanted to remove E4 SD (which is moved to 145 by ABM).
i tried adding `<delete target="145"/> <!-- E4 SD -->` to the deletes section of the custommix file, but that resulted in E4 SD being put back in 106. and doing `<delete target="106"/> <!-- E4 SD -->` didn't give the desired result either.

basically i'm just trying to tidy up my EPG by removing all the HD/SD duplicates

**Comment by AbuBaniaz** [comment]
ABM does not do anything with EPG. 

Have a look at the readme file in the custom folder.
https://github.com/oe-alliance/AutoBouquetsMaker/blob/master/AutoBouquetsMaker/custom/README.txt

An example of usage is here
https://github.com/davesayers2014/CustomMix_Files/blob/master/Standard/sat_282_sky_uk_CustomMix.xml#L116:L155

You don't need the comments, so you can just have
```
<delete target="855"></delete>
```
Bear in mind swaps do not take place if target does not exist or is not HD. Personally, I would a favourite file. Details in the readme file too. 

Please use the forums for support questions

**Comment by chenks** [comment]
yes i know is doesn't so anything with "EPG", but it does create the channel list, yes? which in essence is the EPG in reality?

i've read and looked at both the files you link to, but fail to see where it mentions how to do what i was asking about?

and yes, i know the comments aren't required, they were there just to make it easy to see which channel is being referred to.

Personally i don't think a favourites file is the solution.

i use ABM to do an HD swap, so end up with lots of SD versions of channels that are not wanted on the channel list (some of them right next, other, some not).

so does ABM not have the abilty to remove those SD channels after ABM has swapped them?

and i wasn't aware ABM had a support forum?

**Comment by AbuBaniaz** [comment]
You can take the chip off your shoulder. You've already been blocked from the picon repository because of the tone of your posts and rudeness.

Swap will only take place if both channels exist and target is HD. If you delete one, the criteria is not met.

Favourites.xml will allow you to create the bouquets with only the services you want.

There isn't a support forum for ABM, the same as most other plugins. Use the forum you visit most.
